code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -26,16 +26,15 @@
 		var/mob/living/carbon/human/H = M
 		var/datum/organ/external/affecting = H.get_organ(user.zone_sel.selecting)
 
-		if(affecting.open == 1)
```

Comment:
Instead of removing the check, make it standard with the rest of them (affecting.open == 0) as we may have robots in the future, and trying to fix their robotic internal organs would prove a bit of trouble if they have a robotic chest.